.docker/scripts/init.sh

```diff
@@ -10,6 +10,7 @@ CKAN_USER_EMAIL="${CKAN_USER_EMAIL:-admin@localhost}"
 
 . /app/ckan/default/bin/activate \
   && cd /app/ckan/default/src/ckan \
+  && paster db clean -c /app/ckan/default/production.ini \
```

Comment:
Is this definitely something that will only be run for integration tests? We wouldn't be able to use a non-idempotent action like this in any environment that needs persistence.

Reply:
Yes, that's correct - this `init.sh` script is only for initialising CKAN on the CI test runner container.